validate_yamls.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import os
+import sys
+
+from lib import exception
+from lib.utils import is_package_installed
+from lib.utils import run_command
+
+def validate_package_metadata(package_metadata_file_path):
+
+    try:
+        run_command("yamllint %s" % package_metadata_file_path)
+    except exception.SubprocessError as e:
+        #pylint: disable=no-member
+        print("validation of package metadata file %s failed, output: %s" % (package_metadata_file_path, e.stdout))
+        return
+
+def validate_packages_metadata(base_dir):
+    """
+    Validate specification files of rpm packages built by Open Power Host OS
+    """
+
+    packages = os.listdir(base_dir)
+
+    for package in packages:
+        files = glob.glob("{base_dir}/{package_name}/{package_name}.yaml".format(
+            base_dir=base_dir,package_name=package))
+        for _file in files:
+            validate_package_metadata(_file)
+
+def parse_cli_options():
+    """
+    Parse CLI options
+
+    Returns:
+        Namespace: CLI options. Valid attributes: packages_metadata_base_dir
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-p', '--packages-metadata-base-dir', dest='packages_metadata_base_dir',
+                        required=True, help='packages metadata base directory path')
```

Comment:
Same here.  Perhaps a `--yamls-dir` or just `--base-dir` allows the tool to be more generic.